app/version/version.go

```diff
@@ -40,3 +46,13 @@ func GitCommit() (hash string, timestamp string) {
 
 	return hash, timestamp
 }
+
+// LogCharonInfo logs charon version information along-with the provided message.
+func LogCharonInfo(ctx context.Context, msg string) {
```

Comment:
i couldn't up with a better name, so suggestions welcome

Reply:
`version.LogInfo`

Reply:
"Charon" is superfluous, since everything in this codebase is charon...